lighthouse-core/lib/url-shim.js

```diff
@@ -17,6 +17,13 @@ const Util = require('../report/html/renderer/util.js');
 const URL = /** @type {!Window["URL"]} */ (typeof self !== 'undefined' && self.URL) ||
     require('url').URL;
 
+const tldPlusOneRegex = new RegExp(`\\.(com|co|gov|org|ac|or|edu|go|nic|in|gouv|gob|web|spb|net|` +
```

Comment:
I think we can trim down this list a bit based on the coverage percentages in that sheet, doing just the top 10 already gives us 94% coverage

```
com       2321232
co        1714813
gov       435538
edu       304490
ac        222746
org       202231
go        82772
gob       73721
or        72214
net       43587
----------------
sum       5473344
total     5841097
coverage  ~94%